lib/railway_ipc/persistence/published_message.ex

```diff
@@ -31,7 +31,7 @@ defmodule RailwayIpc.Persistence.PublishedMessage do
     |> validate_required([
       :message_type,
       :encoded_message,
-      :exchange,
+      # :exchange,
```

Comment:
We will need to stop validating presence of exchange for published message records b/c when we publish "re-publish message" commands, we will publish directly to queue with the default, or "', exchange.